src/__internal__/validation-message/validation-message.style.js

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from "styled-components";
+import PropTypes from "prop-types";
+import baseTheme from "../../style/themes/base";
+
+const getColorForValidationMessage = (type, theme) => {
+  switch (type) {
+    case "warning":
+      return theme.colors.warningText;
+    case "info":
+      return theme.colors.info;
+    default:
+      return theme.colors.error;
+  }
+};
+
+const StyledValidationMessage = styled.p`
+  ${({ theme, type }) => css`
+    color: ${getColorForValidationMessage(type, theme)}
+    font-weight: ${type === "error" ? "bold" : "regular"};
+    margin-top: 0px;
```

Comment:
```suggestion
    margin-top: 0px;
```

The px is unnecessary